File: INICIANTE/JAVASCRIPT/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var qtddias = prompt("DIAS");
var anos = parseInt(qtddias / 365)
var meses = parseInt((qtddias % 365) / 30)
var dias = (qtddias % 365) % 30
console.log(anos + " ano(s)")
console.log(meses + " mes(es)")
console.log(dias + " dia(s)")

File: INICIANTE/JAVA/1020.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int dias;
        Scanner sc = new Scanner(System.in);
        dias = sc.nextInt();
        int anos = dias / 365;
        int meses = (dias % 365) / 30;
        int dia = (dias % 365) % 30;
 
        System.out.print(anos + " ano(s)" + "\n");
        System.out.print(meses + " mes(es)" + "\n");
        System.out.print(dia + " dia(s)" + "\n");
    }
} 
